src/scss/components/_button.scss

```diff
@@ -0,0 +1,11 @@
+@use "../variables" as *;
+
+.btn {
+  background-color: $primary;
+  color: $secondary;
+  font-weight: bold;
+  border: 0;
+  border-radius: 2px;
+  padding: 0.75rem 1.15rem;
+  font-size: 1rem;
+}
```

Comment:
- [ ] All the buttons in your project miss the `cursor: pointer;`, as you can see clearly in every website the button should have cursor to be pointer what do you think, Kindly try to add this in your CSS 😄